charts/back8sup/values.yaml

```diff
@@ -2,6 +2,8 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+# kubeVersionOverride -- override what version of Kubernetes to render against
+kubeVersionOverride: ~
```

Comment:
What's the use-case for this variable? If you want to manually set a specific version then `helm template` has the parameter `--kube-version` available for just that.

Reply:
Not sure if argoCD uses --template

Reply:
i changed the way to check for the `apiVersion` looks better now